exercises/concept/stage-heralding/.meta/exemplar.jl

```diff
@@ -10,6 +10,17 @@ const possessive_pronouns = Dict(
     "he/him" => "his"
 )
 
+"""
+    generate_card(line)
+
+Return the generated moderation card from the given `line`.
+
+# Examples
+```jldoctest
+julia> generate_card("»How I learned to say Farewell« – Madeline (she/her). Start: 13:40, Q&A: 14:00, End: 14:05")
+"- Our next speaker is Madeline\n- Her talk is called »How I learned to say Farewell«\n- She will answer your questions in the Q&A session at the end of the talk, starting at 14:00\n\n13:40 - 14:00 - 14:05\n"
+```
```

Comment:
I've included an example here because it's not at all obvious what the moderation card or the line should look like, even if the method expects a certain format.

Reply:
You could do `print(generate_card(...` here for a more readable output?